File: BusRider/Model/JWLineRequest.m
//
//  JWLineRequest.m
//  BusRider
//
//  Created by John Wong on 12/30/14.
//  Copyright (c) 2014 John Wong. All rights reserved.
//

#import "JWLineRequest.h"


@implementation JWLineRequest

- (NSDictionary *)params
{
    return @{
        @"lineId" : self.lineId ?: @""
    };
}

- (NSString *)actionName
{
    return @"line!map2";
}

- (NSString *)validateParams
{
    if (!self.lineId || self.lineId.length <= 0) {
        return @"请先到应用中选择路线";
    }
    return nil;
}

@end

File: BusRider WatchKit Extension/JWLineRequest.swift
//
//  JWLineRequest.swift
//  BusRider
//
//  Created by John Wong on 5/3/15.
//  Copyright (c) 2015 John Wong. All rights reserved.
//

import UIKit

class JWLineRequest: JWRequest {
    
    var lineId = ""
    
    
    override func params() -> Dictionary<NSObject, AnyObject> {
        return [
            "lineId": self.lineId
        ]
    }
    
    override func actionName() -> String {
        return "line!map2"
    }
}
